lms/djangoapps/courseware/tests/test_grades.py

```diff
@@ -146,52 +145,6 @@ def _gradesets_and_errors_for(self, course_id, students):
         return students_to_gradesets, students_to_errors
 
 
-class TestMaxScoresCache(SharedModuleStoreTestCase):
```

Comment:
> Because it was removed in your origin PR. So I have removed it too.

As I previously said, you have to restore it. I've moved it into another place, but you just removed it.